editor/sanitize.js

```diff
@@ -154,17 +153,17 @@ svgedit.sanitize.sanitizeSvg = function(node) {
       var attrName = attr.nodeName;
       var attrLocalName = attr.localName;
       var attrNsURI = attr.namespaceURI;
+
+      // Ignore attributes from custom namespaces and their definitions
+      var splitNS = attrName.match(/([^:]+):(.*$)/);
+      if(splitNS && splitNS[1] in svgedit.ignoredNS) continue;
+      if(splitNS && splitNS[1] == 'xmlns' && splitNS[2] in svgedit.ignoredNS) continue;
```

Comment:
Umh. No.

This is *exactly* the sort of thing I was complaining about.

```
xmlns:se="http://www.w3.org/1999/xhtml" se:onload=..."
```

(and variations on that theme) would now allow a user to enter arbitrary javascript.

You need, *at a minimum*, to check not only that the prefix is one of the keys in the `ignoredNS` hash, but that the *value* is correct.

Better yet (to continue beating a very dead horse) would be an explicit whitelist of elements and attributes. But, if we're going to whitelist a whole namespace, we at least need to do that correctly.